src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -217,32 +217,31 @@ def cli_cosmosdb_collection_delete(client, database_id, collection_id):
 
 def _populate_collection_definition(collection,
                                     partition_key_path=None,
-                                    defaultTtl=None,
+                                    default_ttl=None,
                                     indexing_policy=None):
-    changed = False
+    if not any(not arg is None for arg in [partition_key_path, default_ttl, indexing_policy]):
+        return False
 
-    if partition_key_path:
+    if not partition_key_path is None:
```

Comment:
Please see the most recent change. already fixed.
Also I want the user gets the proper error message and I think azure-cli module should be very thin. So if the provided path doesn't make sense the backend will error and the cli will print out the error message of the backend. I think this would be preferrable over not showing any error message (as we were doing previously) or replicating the arg validation of the backend in the cli.